packages/extension/src/background/handlers/State.ts

```diff
@@ -59,7 +68,11 @@ export default class State {
 
   readonly #authRequests: Record<string, AuthRequest> = {};
 
-  #provider: ProviderInterface | undefined;
+  // Map of providers currently injected in tabs
+  readonly #injectedProviders: Map<chrome.runtime.Port, ProviderInterface> = new Map()
```

Comment:
```suggestion
  readonly #injectedProviders: Map<chrome.runtime.Port, ProviderInterface> = new Map();
```

Weird that the linter didn't complain.